homework.md

```diff
@@ -4,77 +4,76 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
Hi Omar,
good job with the homework, you have one small typo, Question 3 should say: "What is the 2 * 3 * 6 * 1 * 10 * 34?"
Thanks

Reply:
Gret job Omar, 
I like the succinct definition you gave for what Javascript is.